src/services/search.ts

```diff
@@ -0,0 +1,29 @@
+// parent search. holds logic for splitting between different services
+
+import * as play from 'play-dl';
+import { PlayValidation } from '../types/definitions';
+
+// TODO: handle search for other sources. for now, default to yt_videos only
+export const search = async (
```

Comment:
Wrap in class, make static, export the class.
I can foresee this becoming more complicated when search supports more than just YouTube